src/encoded/schemas/biosample.json

```diff
@@ -137,9 +137,12 @@
         }
     },
     "facets": {
-        "biosource_summary": {
+        "biosample_type": {
```

Comment:
When looking at this on local, the biosample_type seems a little redundant with biosource_type, even if they're slightly different - might make sense to remove biosource_type from facets, but up to you.

Reply:
Done.